core/templates/dev/head/combined-tests.spec.ts

```diff
@@ -46,12 +46,14 @@ getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting()
 );
-// Then we find all the tests. Note that known failing files are exempted;
+// Then we find all the tests and controllers,
```

Comment:
Nit: Please wrap lines to 80 chars (not less).

Reply:
Done